migration/v0.36/apply/11-kube-prometheus-stack-crds.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "  - deleting old kube-prometheus-stack CRDs on sc"
+
+      kubectl_do sc delete -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+      kubectl_do sc create -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+    fi
+    if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+      log_info "  - deleting old kube-prometheus-stack CRDs on wc"
+
+      kubectl_do wc delete -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+      kubectl_do wc create -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+    fi
+    ;;
```

Comment:
can we also run the kube-prometheus-stack helm upgrade as part of this script?

Reply:
Done!